core/platform/image/gae_image_services.py

```diff
@@ -79,9 +80,13 @@ def compress_image(image_content, scaling_factor):
                     MAX_RESIZE_DIMENSION_PX, float(max(width, height))))
             new_width = int(width * new_scaling_factor)
             new_height = int(height * new_scaling_factor)
-        return images.resize(
-            image_data=image_content,
-            width=min(new_width, MAX_RESIZE_DIMENSION_PX),
-            height=min(new_height, MAX_RESIZE_DIMENSION_PX))
+        maxsize = (min(new_width, MAX_RESIZE_DIMENSION_PX),
```

Comment:
new_image_dimensions or compressed_image_dimensions.

(Issues with the existing name: "max size" is two words; also it is not clear what the "maximum" here is relative to.)

Also, break after the first '('. This was already mentioned in the last review comment: https://github.com/oppia/oppia/pull/9828#discussion_r451292624

Reply:
Done